### DISCUSSION TITLE
Share Favorites

### DISCUSSION THREAD

**Discussion by jacob-daniel-waller** [ente-io/ente#3548] [Enhancements]
It appears you can only share single photos or albums. It would be awesome to share all the photos I favorite too. I can work around this and create an album called Favorites and add them to that but that seems silly. Why not allow your favorites to be shared like an album. Trying to do this from iOS

**Comment by KestrelWild** [comment]
I was just creating a new request for this! 

I want to be able to share a LOT of photos with family, but I don't really want to spend time putting them all in albums. Also, there are no hierarchical albums - the best way to share using albums would be if you could share one top-level album that included a bunch of sub-albums.

BUT, the lazy man's option is to just share Favorites with people. Then you just mark your favorite photos and boom, they show up there for family to see. They can search by date, people, places, etc and  scroll through them all, etc.

**Comment by vishnukvmd** [comment]
This is now available starting v1.1: https://ente.io/blog/photos-v1.1/
